test/functionality/_helper/dataflowgraph-builder.ts

```diff
@@ -0,0 +1,46 @@
+import type { DataflowGraphEdgeAttribute, NodeId, REnvironmentInformation } from '../../../src'
+import { DataflowGraph, EdgeType } from '../../../src'
+import { LocalScope } from '../../../src/dataflow/environments/scopes'
+
+export function emptyGraph() {
+	return new DataflowGraphBuilder
```

Comment:
I thought i had an eslint rule disallowing constructor calls without parenthesis (now tracked in https://github.com/Code-Inspect/flowr/issues/667).

Reply:
```suggestion
	return new DataflowGraphBuilder()
```